docs/src/main/sphinx/sql/set-session.rst

```diff
@@ -28,15 +49,14 @@ generation::
 
     SET SESSION optimize_hash_generation = true;
 
-An example of the usage of catalog session properties is the :doc:`Accumulo
-connector </connector/accumulo>`, which supports a property named
-``optimize_locality_enabled``. If your Accumulo catalog is named ``data``, you
-would use the following to set the catalog session property::
+The following example sets the ``optimize_locality_enabled`` catalog session
+property for an :doc:`Accumulo catalog </connector/accumulo>` named ``data``::
```

Comment:
I know this is late to the party, and I know this was inherited, but "data" is a terrible name for a catalog. That name is designed for ambiguity.
Can we make it "accum01" or some such to relate it to accumulo but also show that there can be more than one?

Reply:
Done